### PR TITLE
Sia2 kwargs

### PR DESCRIPTION
**Pull request by andamian**
#221

### CODE REVIEW

**Comment by cbanek** [comment]
Guessing the build break is some kind of upstream future votable thing in astropy?

**Comment by tomdonaldson** [comment]
> Guessing the build break is some kind of upstream future votable thing in astropy?

Yes, @cbanek .  Surely this is due to https://github.com/astropy/astropy/pull/9505, which will be released soon with astropy 4.1.

Whether these failures get fixed here to pass with both `byte` and `str` values, we should probably think about whether we have other coordination to do with respect to that change.  I.e., which pyvo versions should pass testing with both astropy 4.0 and 4.1?

**Comment by andamian** [comment]
All green now! @funbaker do you have time to take a look? Thanks

File: pyvo/dal/sia2.py [hunk 1]
```diff
@@ -305,6 +312,16 @@ def __init__(self, url, pos=None, band=None, time=None, pol=None,
         for rf in _tolist(res_format):
             self.res_format.add(rf)
 
+        for ii in kwargs:
```
Comment (funbaker) [inline pyvo/dal/sia2.py:315]: `for name, value in kwargs.items()` would be more readable

**Comment by andamian** [comment]
@funbaker - anything else needed for the approval?

**Pull request merged**